app/src/main/java/pl/edu/pjatk/goodwill_ninjas/blooddonor_android/api/userFeat/UserFeatResponse.kt

```diff
@@ -6,7 +6,16 @@ data class UserFeatResponse(
     @JsonProperty("userId") val userId: Int = 0,
     @JsonProperty("featId") val featId: String = "",
     @JsonProperty("featName") val featName: String = "",
-    @JsonProperty("featDescription") val featDescription: String = "",
-    @JsonProperty("achievedRanks") val achievedRanks: String = "",
-    @JsonProperty("nextRanks") val nextRanks: String = ""
+    @JsonProperty("featDescription") val featDescription: String? = "",
+    @JsonProperty("achievedRanks") val achievedRanks: List<UserFeatRanks> = listOf(UserFeatRanks()),
+    @JsonProperty("nextRanks") val nextRanks: List<UserFeatRanks> = listOf(UserFeatRanks())
```

Comment:
These should be in snake case coming from the API. Want me to fix that or are we sticking with this for the time being?